source/main/MainMenu.h

```diff
@@ -33,7 +33,7 @@
 
 #include <map>
 #include <OgreFrameListener.h>
-#include <OgreWindowEventUtilities.h>
+#include <Bites/OgreWindowEventUtilities.h>
```

Comment:
this and following: you should not need to add the Component folder prefix here. The include path should include the subdirectory.